content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -45,71 +156,46 @@ team. These credentials are only used to generate the
 [authorization](#authorization) URL DVC will later prompt to visit in order to
 connect to the Google Drive.
 
-> Note that Google Drive API has usage limits/quotas per credentials in a
-> _project_ (which you can review in the
-> [OAuth consent screen](https://console.developers.google.com/apis/credentials/consent)).
-> Please keep this in mind when sharing them, or you may
-> [exceed the limits](https://developers.google.com/drive/api/v2/handle-errors?hl=ro#resolve_a_403_error_usage_limit_exceeded).
-
-## URL format for Google Drive remotes
-
-A GDrive DVC remote URLs is needed for the `dvc remote add` command. It can be
-constructed with a _base_, and an **optional** _path_ to a folder i.e.
-`gdrive://<base>/path/to/folder`. The base can be one of:
-
-1. `root` - indicates your topmost Google Drive directory.
+Finally, use the `dvc remote modify` command to set the credentials (for each
+GDrive remote), for example:
 
 ```dvc
-$ dvc remote add -d mygdroot gdrive://root
-$ dvc remote add mygdfolder gdrive://root/path/to/folder
+$ dvc remote add gdfolder gdrive://0AIac4JZqHhKmUk9PDA
+$ dvc remote modify gdfolder gdrive_client_id <client ID>
+$ dvc remote modify gdfolder gdrive_client_secret <client secret>
 ```
 
-2. Google Drive _Folder ID_
-
-To obtain the unique _Folder ID_ in question, navigate into that folder in your
-web browser, and find it in the address bar. For example, for
-`https://drive.google.com/drive/folders/0AIac4JZqHhKmUk9PDA`:
-
-```dvc
-$ dvc remote add mygdfolder gdrive://0AIac4JZqHhKmUk9PDA
-$ dvc remote add mygdsubfolder \
-                 gdrive://0AIac4JZqHhKmUk9PDA/sub/path
-```
+> Note that Google Drive API usage limits/quotas apply per _project_ client and
+> can be reviewed in the
+> [OAuth consent screen](https://console.developers.google.com/apis/credentials/consent).
+> Please keep this in mind when sharing them, or you may
+> [exceed the limits](https://developers.google.com/drive/api/v2/handle-errors?hl=ro#resolve_a_403_error_usage_limit_exceeded).
 
-[Shared drives](https://developers.google.com/drive/api/v2/about-shareddrives)
-should be referenced by _Folder ID_ in the same way.
+## Using service accounts
 
-3. `appDataFolder` - special
-   [application-specific data](https://developers.google.com/drive/api/v2/appdata)
-   folder only accessible by your application and hidden from the user.
+A [service account](https://cloud.google.com/iam/docs/service-accounts) is a
+Google account associated with your GCP project, and not a specific user. They
+are intended for scenarios where your code needs to access data on its own, e.g.
+running inside a Compute Engine, automatic CI/CD, etc. No interactive user OAuth
+authentication is needed.
 
-```dvc
-$ dvc remote add mygdappata gdrive://appDataFolder
-```
+> This requires having your own
+> [GC project](/doc/user-guide/setup-google-drive-remote#connect-using-a-custom-google-cloud-project)
```

Comment:
link is probably broken - please check other places

Reply:
I've been keeping them current.